File: public/examples/integrations/toolkits/search/google_finance/get_stock_historical_data_example_call_tool.js
import { Arcade } from "@arcadeai/arcadejs";

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const USER_ID = "you@example.com";
const TOOL_NAME = "GoogleFinance.GetStockHistoricalData";

const toolInput = {
	ticker_symbol: "GOOG",
	exchange_identifier: "NASDAQ",
	window: "ONE_MONTH",
};

const response = await client.tools.execute({
	tool_name: TOOL_NAME,
	input: toolInput,
	user_id: USER_ID,
});

console.log(response);

File: public/examples/integrations/toolkits/search/google_finance/get_stock_historical_data_example_call_tool.py
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

USER_ID = "{arcade_user_id}"
TOOL_NAME = "GoogleFinance.GetStockHistoricalData"

tool_input = {
    "ticker_symbol": "GOOG",
    "exchange_identifier": "NASDAQ",
    "window": "ONE_MONTH",
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
    user_id=USER_ID,
)
print(response)
